CorsixTH/Lua/objects/helicopter.lua

```diff
@@ -62,6 +66,12 @@ function Helicopter:tick()
     self:setSpeed(0, 0)
     self.spawned_patients = 0
   elseif phase == 85 then
+    if not self.announced then
+      local ui = TheApp.ui
+      ui:playAnnouncement(ui.hospital.emergency.disease.emergency_sound, AnnouncementPriority.Critical)
+      ui.adviser:say(_A.information.emergency)
+      self.announced = true
+    end
```

Comment:
While I said I would be fine with tying it to the helicopter, my only query is whether tying it into phase is the correct thing?  Should the helicopter be deciding to trigger these events?
If for some reason in future we used the helicopter for something else then this would trigger the emergency texts.

Reply:
We can define that later, add a helicopter.type when we know what the other use is.